core/domain/story_jobs_one_off.py

```diff
@@ -94,3 +94,55 @@ def reduce(key, values):
                 sum(ast.literal_eval(v) for v in values))])
         else:
             yield (key, values)
+
+
+class UpdateStoryThumbnailSizeOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job to update the thumbnail_size_in_bytes in story models. """
+
+    _DELETED_KEY = 'story_deleted'
+    _ERROR_KEY = 'update_error'
+    _SUCCESS_KEY = 'thumbnail_size_updated'
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [story_models.StoryModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        story = story_fetchers.get_story_by_id(item.id)
+
+        try:
+            # We are not updating thumbnail_filename here, but using it call the
+            # update for story thumbnail_size_in_bytes.
+            # old_value and new_value are same here, because the update for
+            # thumbnail_size_in_bytes is called from within the code for
+            # updating thumbnail_filename in story_services.py file.
+            commit_cmds = [story_domain.StoryChange({
+                'cmd': story_domain.CMD_UPDATE_STORY_PROPERTY,
+                'property_name': (
+                    story_domain.STORY_PROPERTY_THUMBNAIL_FILENAME),
+                'new_value': story.thumbnail_filename,
+                'old_value': story.thumbnail_filename
+            })]
+
+            story_services.update_story(
+                feconf.MIGRATION_BOT_USERNAME, item.id, commit_cmds,
+                'Update story thumbnail size'
+            )
+            yield (UpdateStoryThumbnailSizeOneOffJob._SUCCESS_KEY, 1)
+        except Exception as e:
+            logging.exception(
+                'Updating story thumbnail_size_in_bytes %s failed validation'
```

Comment:
Why is the exception saying that the update "failed validation"? Since you're catching a general exception here, how do we know that its cause is a validation failure?

Reply:
From the `update_story` function inside the `Story_Services`, it says only `ValidationError` in the comments, but it might throws `ValueError` if no commit message is present, else it's always `ValidationError`.

Reply:
But here you have a commit message, don't you -- 'Update story thumbnail size'?

That said, see @srikanthkb's point about not needing an actual update here, since this is more of a change to the schema.